resume.css

```diff
@@ -79,6 +79,10 @@ h1 + ul + p {
         box-shadow: none;
         -webkit-box-shadow: none;
     }
+    /* Do not underline abbr tags in PDF */
```

Comment:
The underline is unnecessary and makes the example resume look cluttered. Please change the CSS so this directive applies to all formats.

Reply:
For a11y reasons, I would advise to keep abbr underlined in web page. Otherwise, it is not visible to see that this term has an explanation by the `abbr` tag.

Maybe you have one, but I don't see any reason to make `abbr` tags look like standard text.

See [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr) for more informations.

After theses considerations, what are your toughts?

Reply:
In addition, those who do not use `abbr` won't see any change.

And those who do are probably expecting their `abbr` to be rendered differently.

Reply:
This is a tricky one.

I want users who want the conventional look of a print resume to be able to ignore the CSS, i.e. "looks like a print resume" should be the default. And I'm a little worried that the dotted underline in the default resume will be confusing or off-putting for people who don't know CSS or HTML.

But the abbr feature is useful. So how about this?

    keep the CSS how you have it, i.e. browser default rendering for browser view, and no underline for PDF
    per MDN, add font-variant: none to the PDF style (apparently some browsers use small caps, but we want the most vanilla, boring typography possible)
    remove the example usage of abbr from resume.md
    in the last paragraph of the README, mention that the abbreviations extension is enabled (and link to https://python-markdown.github.io/extensions/abbreviations/).

If this sounds good, feel free to update the PR. If you prefer, I can handle the last bullet (the README) after we land your change.

Reply:
I handled all your requests.